### DISCUSSION TITLE
Question about import modules

### DISCUSSION THREAD

**Discussion by alemscoder** [esm-dev/esm.sh#747] [Q&A] [answered]
Is there any way to import the modules and have them served without the build process? I mean like a `?raw` form but maintaining the resolution of dependencies. I'm having this problem in this [case](https://codepen.io/alemscoder/pen/qBgBRWM), import more than one ES module from the same library results in shared modules being bundled. Thanks in advace

**Comment by renhiyama** [comment]
> Is there any way to import the modules and have them served without the build process? I mean like a `?raw` form but maintaining the resolution of dependencies. I'm having this problem in this [case](https://codepen.io/alemscoder/pen/qBgBRWM), import more than one ES module from the same library results in shared modules being bundled. Thanks in advace
> 

Any link that you would suggest? And how do you think the final response code should be?

**Comment by alemscoder** [comment]
[Link1](https://esm.sh/@material/web@1.0.1/switch/switch.js)            -> this link return the module and deps bundled
[Link2](https://esm.sh/@material/web@1.0.1/switch/switch.js?raw)    -> this return the module in raw format, no bundled, but have original 
                                                                                                                   bare modules and can not be loaded in the browser
Check this example in Skypack, something like this, code not bundled and resolve bare modules
[Link3](https://cdn.skypack.dev/@material/web@1.0.0/switch/switch.js)
[Link4](https://cdn.skypack.dev/-/@material/web@v1.0.0-ENxXCb4aye6TXKBblhb1/dist=es2019,mode=imports/unoptimized/switch/switch.js)
[Codepen](https://codepen.io/alemscoder/pen/qBgBRWM)

**Comment by alemscoder** [comment, marked as answer]
This problem was solved using `?no-bundle` flag added in the release V135. Thank you @ije
